src/options/lint.rs

```diff
@@ -0,0 +1,9 @@
+use clap::Parser;
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Serialize, Deserialize, Parser)]
+pub struct LintOpts {
+    /// If the lint should be run and compared to the most recently published schema
```

Comment:
Could use @StephenBarlow's help on this help text. It's _close_ but still a bit confusing. Maybe something like this? 

```suggestion
    /// Ignore existing lint violations for a published subgraph. If passed, the command will only report lint violations introduced by recent changes.
```

Reply:
Agreed, any comment and documentation suggestions are greatly appreciated